homeassistant/components/media_player/xiaomi_tv.py

```diff
@@ -0,0 +1,105 @@
+"""
+This platform adds support for Xiaomi TVs
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/xiaomi_tv/
+"""
+
+import logging
+import homeassistant.helpers.config_validation as cv
+import voluptuous as vol
+from homeassistant.const import (CONF_HOST, CONF_NAME, STATE_OFF, STATE_ON)
+from homeassistant.components.media_player import (
+    SUPPORT_TURN_ON, SUPPORT_TURN_OFF, MediaPlayerDevice, PLATFORM_SCHEMA,
+    SUPPORT_VOLUME_STEP)
+
+REQUIREMENTS = ['pymitv==1.0.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_XIAOMI_TV = SUPPORT_VOLUME_STEP | SUPPORT_TURN_ON | \
+                    SUPPORT_TURN_OFF
+
+"""No host is needed for configuration, however it can be set."""
```

Comment:
If I'm correct, this should be a comment (`#`), not a docstring.

Reply:
You're probably right. However, when looking at other platforms, they seem to be using exclusively docstring.

Reply:
Oh sorry then, wasn't aware of that.

Reply:
I did look at the PEP8 style guide, and it says:
> Block comments generally apply to some (or all) code that follows them, and are indented to the same level as that code. Each line of a block comment starts with a # and a single space (unless it is indented text inside the comment).
So I'd say you're right.